extensions/interactions/LogicProof/LogicProof.html

```diff
@@ -18,9 +18,10 @@
         <textarea ng-trim="false" rows="10"></textarea>
       </div>
     </div>
-    <button class="btn btn-default" ng-click="submitProof()">Submit</button>
+    <!-- The onclick for proof submission is added in the javascript -->
+    <button class="btn btn-default logic-proof-submit">Submit</button>
     <button class="btn btn-default" ng-click="showHelp()">Possible lines</button>
-    <p class="logic-proof-mistake-message"> <[proofError]>  </p>
+    <p class="logic-proof-mistake-message"> <[mistakeMessage]>  </p>
```

Comment:
I'm not finding this intuitive at all, I'm afraid. It's also not very clean, and sometimes gets covered up by the fallback message that appears in the bottom right.

I thought about the design a little bit -- would it be possible to do something along the following lines? https://codemirror.net/demo/lint.html